Sources/StreamChatTestTools/Models/Attachments/ChatMessageAttachmentEnvelope_Mock.swift

```diff
@@ -5,7 +5,7 @@
 import Foundation
 @testable import StreamChat
 
-public extension ChatMessageAttachmentEnvelope {
+public extension AnyAttachmentPayload {
```

Comment:
nit: `ChatMessageAttachmentEnvelope_Mock` -> `AnyAttachmentPayload_Mock`

Reply:
fixed in 2c405cf, thanks!